src/app/sharedUI/molecules/ScrollableScreensView/Context.ts

```diff
@@ -0,0 +1,19 @@
+import { createContext, useContext } from "react"
+import { ScrollableScreenEntity } from "./types"
+
+export interface ScrollableScreensContextValue {
+  activeScreenIndex: number
+  activeScreen: ScrollableScreenEntity
+}
+
+export const ScrollableScreensContext = createContext(null as any)
```

Comment:
nice. thanks for doing it this way! if you want, that works too

```suggestion
export const ScrollableScreensContext = createContext(null!)
```

Reply:
If we do this, then this warning is displayed

<img width="1289" alt="image" src="https://user-images.githubusercontent.com/3513494/178713127-df836780-60a6-4137-a211-82ac7ea5245a.png">

Reply:
weird. ok cool